src/sssom/context.py

```diff
@@ -86,6 +86,8 @@ def add_built_in_prefixes_to_prefix_map(
                 raise ValueError(
                     f"Built-in prefix {k} is specified ({prefix_map[k]}) but differs from default ({builtinmap[k]})"
                 )
+    if "dc" in prefix_map and "dcterms" in prefix_map and prefix_map["dc"] == prefix_map["dcterms"]:
+        del prefix_map["dc"]
```

Comment:
I dont know who wrote this but this seems misplaced. Why is this here?

Reply:
See also https://github.com/mapping-commons/sssom/pull/302 for a potential upstream fix. I don't think this should be a blocker for this PR, since prefix map loading isn't very principled in sssom-py already, I don't see any harm in adding some additional specific hacks. A future PR can switch over to using `curies.Converter` natively to handle prefix map loading and support synonyms.

Reply:
Yea, these entries should not exist in the sssom schema context. We want to move away from them entirely. 

I would prefer to wait for the ssssom schema release and then get rid of this dc superfluous code..

Reply:
This is for now necessary because the LinkML context generator automatically adds those prefixes.

Reply:
We actually fixed this the right way with the SSSOM Schema v0.14, I removed related code for checking/fixing DC/DCTERMS in e24fec2.